cfme/tests/physical_infrastructure/test_physical_rack_details.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+import pytest
+
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.physical.provider.lenovo import LenovoProvider
+
+pytestmark = [pytest.mark.tier(3), pytest.mark.provider([LenovoProvider], scope="function")]
+
+
+@pytest.fixture(scope="module")
+def physical_rack(appliance, provider, setup_provider_modscope):
+    # Get and return the first physical rack
+    physical_racks = appliance.collections.physical_racks.filter({"provider": provider}).all()
+    return physical_racks[0]
```

Comment:
This will raise an IndexError if it doesn't find any racks, this should be an explicit `pytest.skip`, as we don't have the resources available to run the test.

The fixture should also use `setup_provider`, not the modscope function. The provider itself is function scoped, and the fixture will control add/remove to do so as little as possible.